src/pal/src/thread/process.cpp

```diff
@@ -61,6 +61,24 @@ SET_DEFAULT_DEBUG_CHANNEL(PROCESS); // some headers have code with asserts, so d
 #include <stdint.h>
 #include <dlfcn.h>
 
+#ifdef __linux__
+#include <sys/syscall.h> // __NR_membarrier
+// Ensure __NR_membarrier is defined for portable builds.
+# if !defined(__NR_membarrier)
+#  if defined(__amd64__)
+#   define __NR_membarrier  324
+#  elif defined(__i386__)
+#   define __NR_membarrier  375
+#  elif defined(__arm__)
+#   define __NR_membarrier  389
+#  elif defined(__aarch64__)
+#   define __NR_membarrier  283
+#  elif
```

Comment:
I took these numbers from: https://fedora.juszkiewicz.com.pl/syscalls.html.
The table is auto generated based on kernel source. On amd64 and i386 they match what I have in my headers. I haven't verified the arm and arm64 values.

Reply:
I can confirm the arm64 value, it matches what I can see in the headers of the rootfs that we use for the lab builds.